### ISSUE TITLE
vscode.dev can not use Simple Browser.

### ISSUE THREAD

**Issue by aleleba** [microsoft/vscode-remote-release#10023]
vscode.dev can not use Simple Browser.

The problem: when try to navigate inside vscode.dev in Simple Browser, all webpages refuses connection

- VSCode Version: 1.90.2
- Local OS Version: 1.90.2
- Remote OS Version: 1.90.2
- Remote Extension/Connection Type: Tunnel

- Logs:

Steps to Reproduce:

1. Ctrl or Cmd + Shift + P
2. Search for Simple Browser
3. Try to navigate and connection refused

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: No

**Comment by aleleba** [comment]
Do someone know if this is the right place to put this issue? 👍🏼  thank you in advance.

**Comment by TechnoFairyGirl** [comment]
> Do someone know if this is the right place to put this issue? 👍🏼 thank you in advance.

I doubt it. This is for running VSCode on remote machines (e.g. via SSH).

**Comment by aleleba** [comment]
No with SSH, but yes on remote machines, Simple browser is not working in vscode.dev when connecting to a dev tunnel.

**Comment by connor4312** [comment]
Guessing this has to do with the security policies on vscode.dev

**Comment by aleleba** [comment]
> Guessing this has to do with the security policies on vscode.dev (can be repro'd without any remote connection on the vscode.dev homepage)

Yes, it happened because the security policies on vscode.dev, we can not use simple browser inside vscode.dev only on local it works 👍🏼

**Comment by joaomoreno** [comment]
@mjbvz We should disable Simple Browser in unsupported scenarios (ie serverless web).

**Comment by mjbvz** [comment]
It should actually works if the page sets these headers:

```
res.setHeader('Cross-Origin-Resource-Policy', 'cross-origin');
res.setHeader('Cross-Origin-Embedder-Policy', 'require-corp');
```

The problem is that most servers do not set these. As a workaround for local development, you can make sure your dev server sets them

Afaik there's not a good way to workaround this besides the [`credentialless` attribute](https://developer.chrome.com/blog/iframe-credentialless/), but that unfortunately when I last tested it seems to break our web views. I can revisit this at some point. That still won't fix the simple browser completely tho as many public sites like YouTube simple do not allow themselves to be shown in iframes

**Comment by mjbvz** [comment]
Closing in favor of https://github.com/microsoft/vscode/issues/228522 since it happens in core too
